frigate/data_processing/real_time/license_plate.py

```diff
@@ -50,6 +53,22 @@ def process_frame(
     def handle_request(self, topic, request_data) -> dict[str, any] | None:
         return
 
-    def expire_object(self, object_id: str):
+    def expire_object(self, object_id: str, camera: str):
         if object_id in self.detected_license_plates:
             self.detected_license_plates.pop(object_id)
+
+            if object_id in self.camera_current_cars.get(camera, []):
+                self.camera_current_cars[camera].remove(object_id)
+
+                if len(self.camera_current_cars[camera]) == 0:
+                    self.requestor.send_data(
+                        "tracked_object_update",
+                        json.dumps(
+                            {
+                                "type": TrackedObjectUpdateTypesEnum.face,
```

Comment:
should be `TrackedObjectUpdateTypesEnum.lpr` and you probably also need `plate` and any other attributes here